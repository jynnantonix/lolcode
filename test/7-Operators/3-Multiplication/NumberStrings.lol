This test makes sure the multiplication operator returns an error on strings containing integer and/or float arguments.

HAI 1.2
	VISIBLE PRODUKT OF "1" AN "2.75" MKAY?
KTHXBAI


