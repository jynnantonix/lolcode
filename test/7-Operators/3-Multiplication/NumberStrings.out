Unexpected error: can't multiply sequence by non-int of type 'str'
